perma_web/perma/forms.py

```diff
@@ -75,6 +75,41 @@ class Meta:
     def clean_registrar(self):
         registrar = self.cleaned_data["registrar"]
         return registrar
+        
+        
+class CreateUserFormWithCourt(CreateUserForm):
+    """
+    add court to the create user form
+    """
+
+    requested_account_note = forms.CharField(required=True)
+    
+    class Meta:
+        model = LinkUser
+        fields = ["first_name", "last_name", "email", "requested_account_note"]
+
+    def __init__(self, *args, **kwargs):
+        super(CreateUserFormWithCourt, self).__init__(*args, **kwargs)
+        self.fields['requested_account_note'].label = "Your court"
+        self.fields['first_name'].label = "Your first name"
+        self.fields['last_name'].label = "Your last name"
+        self.fields['email'].label = "Your email"
+        
+        
+class CreateUserFormWithUniversity(CreateUserForm):
```

Comment:
Couple notes on this class:

- docstring should be updated
- seems very similar to CreateUserFormWithCourt -- maybe it should derive from that one or vice versa?
- it's not clear why we're changing the labels for first_name, last_name, and email in CreateUserFormWithCourt but not the other three -- maybe add a comment if that's a special case?